gc/base/gcutils.cpp

```diff
@@ -209,6 +213,8 @@ getExpandReasonAsString(ExpandReason reason)
 		return "insufficient free space following gc";
 	case SCAV_RATIO_TOO_HIGH:
 		return "excessive time being spent scavenging";
+	case GC_HIGH_SPACE_LOW:
+		return "space too low and gc ratio too high";	
```

Comment:
Align the names of those too opposite constants. For example: 

FREE_SPACE_HIGH/LOW_AND_GC_TIME_LOW/HIGH

I wonder if AND or OR is more appropriate!?

Reply:
I think given their usage, it might make more sense to use "OR" in those 2 cases, since "AND" will not always be the case. 

ie, gc ratio might be too high, but free space could be fine, suggesting expansion